src/comm.jl

```diff
@@ -797,7 +819,7 @@ function commSBN(domain::Domain, fields)
 
    myRank = MPI.Comm_rank(comm)
 
-   if planeMin || planeMax
+   if planeMin | planeMax
```

Comment:
This should not matter?

Reply:
No. We concluded the same. But at least we won't have to debate it ever again :-) .